conf/modules/annotate_svs.config

```diff
@@ -42,15 +42,15 @@ process {
         publishDir = [
             path: { "${params.outdir}/svs/family/${meta.id}" },
             mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            saveAs: { filename -> filename.equals('versions.yml') || !params.skip_rank_variants ? null : filename }
```

Comment:
Why do you skip saving vep output when skip_rank_variants?

Reply:
I skip saving vep output when **not** skipping rank variants. 

I opted for only having one output file. So the default output will be
```
{family_id}_svs_cnvs_merged_annotated_ranked.vcf.gz
```
 with ranked variants output from genmod within `RANK_VARIANTS_SVS`.

If ranking is off, the output file will be:
```
{family_id}_svs_cnvs_merged_annotated.vcf.gz
```
 from VEP within `ANNOTATE_SVS` with just annotated variants.